python/ray/_private/runtime_env.py

```diff
@@ -24,6 +26,41 @@
 FILE_SIZE_LIMIT = 50 * 1024 * 1024  # 50MB
 
 
+class RuntimeEnvDict:
+    """Parses and validates runtime env dictionary from the user."""
+
+    def __init__(self, runtime_env_json: dict):
+        if "conda" in runtime_env_json:
+            self.conda = runtime_env_json["conda"]
+        else:
+            self.conda = None
+        if "files" in runtime_env_json:
+            self.files = runtime_env_json["files"]
+        else:
+            self.files = None
+        # TODO(ekl) we should have better schema validation here.
+
+    def to_worker_env_vars(self, override_environment_variables: dict) -> dict:
+        """Given existing worker env vars, return an updated dict.
+
+        This sets any necessary env vars to setup the runtime env.
+        TODO(ekl): env vars is probably not the right long term impl.
```

Comment:
yeah let's also just put `env_vars` into runtime_env, maybe in separate PR? up to you

Reply:
Added as a TODO for now